client/src/components/Lyrics.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {LyricsSection, LyricsWrapper, SongTitle, Status, Paragraph} from './LyricsStyles'
+import './Header.css'
+
+
+const Lyrics = () => {
+  return (
+    <LyricsSection>
+      <LyricsWrapper>
+        <Status>Translation</Status>
+        <SongTitle>Song</SongTitle>
+        <Paragraph>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. 
```

Comment:
this is hard coded. think that you get a single text with all the lyrics - and you need to present them.
build the component to handle that